plaidbench.py

```diff
@@ -98,9 +100,20 @@ def main():
     if args.fp16:
         from keras.backend.common import set_floatx
         set_floatx('float16')
+
     batch_size = int(args.batch_size)
-    truncation_size = 64 / batch_size
-    epoch_size = truncation_size * batch_size
+    epochs = args.epochs
+    examples = args.examples
+    epoch_size = examples / epochs
+
+    if epochs > examples:
```

Comment:
Do a `raise ValueError("Epochs must be smaller than examples and evenly divide it.")`